src/app/services/services.module.ts

```diff
@@ -0,0 +1,126 @@
+import { ElectronRendererMessagingService } from "jslib-electron/services/electronRendererMessaging.service";
+import { ElectronRendererSecureStorageService } from "jslib-electron/services/electronRendererSecureStorage.service";
+import { ElectronRendererStorageService } from "jslib-electron/services/electronRendererStorage.service";
+
+import { Account } from "../../models/account";
+import { I18nService } from "../../services/i18n.service";
+import { LoginGuardService } from "../../services/loginGuard.service";
+import { NativeMessagingService } from "../../services/nativeMessaging.service";
+import { PasswordRepromptService } from "../../services/passwordReprompt.service";
+import { StateService } from "../../services/state.service";
+import { SearchBarService } from "../layout/search/search-bar.service";
+
+import { InitService } from "./init.service";
+
+@NgModule({
+  imports: [JslibServicesModule],
+  declarations: [],
+  providers: [
+    InitService,
+    NativeMessagingService,
+    SearchBarService,
+    LoginGuardService,
+    {
+      provide: APP_INITIALIZER,
+      useFactory: (initService: InitService) => initService.init(),
+      deps: [InitService],
+      multi: true,
+    },
+    {
+      provide: STATE_FACTORY,
+      useValue: new StateFactory(GlobalState, Account),
+    },
+    {
+      provide: CLIENT_TYPE,
+      useValue: ClientType.Desktop,
+    },
+    { provide: LogServiceAbstraction, useClass: ElectronLogService, deps: [] },
+    {
+      provide: PlatformUtilsServiceAbstraction,
+      useClass: ElectronPlatformUtilsService,
+      deps: [
+        I18nServiceAbstraction,
+        MessagingServiceAbstraction,
+        CLIENT_TYPE,
+        StateServiceAbstraction,
+      ],
+    },
+    {
+      provide: I18nServiceAbstraction,
+      useFactory: (window: Window) => new I18nService(window.navigator.language, "./locales"),
+      deps: [WINDOW],
+    },
+    {
+      provide: MessagingServiceAbstraction,
+      useClass: ElectronRendererMessagingService,
+      deps: [BroadcasterServiceAbstraction],
+    },
+    { provide: StorageServiceAbstraction, useClass: ElectronRendererStorageService },
+    { provide: SECURE_STORAGE, useClass: ElectronRendererSecureStorageService },
+    {
+      provide: CryptoServiceAbstraction,
+      useClass: ElectronCryptoService,
+      deps: [
+        CryptoFunctionServiceAbstraction,
+        PlatformUtilsServiceAbstraction,
+        LogServiceAbstraction,
+        StateServiceAbstraction,
+      ],
+    },
+    {
+      provide: SystemServiceAbstraction,
+      useFactory: (
+        messagingService: MessagingServiceAbstraction,
+        platformUtilsService: PlatformUtilsServiceAbstraction,
+        stateService: StateServiceAbstraction
+      ) => new SystemService(messagingService, platformUtilsService, null, stateService),
+      deps: [MessagingServiceAbstraction, PlatformUtilsServiceAbstraction, StateServiceAbstraction],
+    },
```

Comment:
Did we forget to change this from `useFactory`?

Reply:
Missed this one - done. Put the token declaration in this file (not jslib) because it's not used by any other `services.module.ts`. (only in browser, and that's in the background)